demos/taco-nft-demo/src/NFTConditionBuilder.tsx

```diff
@@ -0,0 +1,122 @@
+import { conditions } from '@nucypher/taco';
+import { useEthers } from '@usedapp/core';
+import React, { useState } from 'react';
+
+interface Props {
+  condition?: conditions.Condition | undefined;
+  setConditions: (value: conditions.Condition) => void;
+  enabled: boolean;
+}
+
+export const NFTConditionBuilder = ({
+  condition,
+  setConditions,
+  enabled,
+}: Props) => {
+  const { library } = useEthers();
+  const goerliNFTAddress = '0x932Ca55B9Ef0b3094E8Fa82435b3b4c50d713043'; // https://goerli-nfts.vercel.app/
```

Comment:
Tapir is going to use sepolia (lynx will still use goerli) - so we'll need a new NFT address on sepolia. 

The both will still use Mumbai, of if using an NFT on mumbai is simpler then that works too.

(cc @KPrasch )

Reply:
I will update the demos when we release TACo on Sepolia. I always have to test them in cases like this anyway.